src/django_components/component.py

```diff
@@ -618,7 +629,11 @@ def _render_impl(
         self._validate_inputs()
 
         context_data = self.get_context_data(*args, **kwargs)
-        self._validate_outputs(context_data)
+        self._validate_outputs(data=context_data)
+
+        # Process JS and CSS files
+        cache_inlined_js(self.__class__, self.js or "")
+        cache_inlined_css(self.__class__, self.css or "")
```

Comment:
This caches the Component's inlined JS and CSS, so it will be retrievable from within [the endpoint that serves components' inlined JS / CSS](https://github.com/EmilStenstrom/django-components/pull/688/files#diff-371d604ce82d2a01458065f1b137fd981b12af1b0b5c5fb2c1f7f5b45c20003fR699).

However, this will make more sense in the following MRs. Because in this MR we don't post-process the inlined JS / CSS in any way.